source/products/hansemerkur/cancer-insurance/webapp.rst

```diff
@@ -0,0 +1,19 @@
+Webapp
+======
+
+Example Kasko JS embed code
+---------------------------
+
+.. code:: html
+
+    <script type="text/javascript" src="https://js.kasko.io"></script>
+    <div id="kasko-widget-container"></div>
+    <script>
+      Kasko.Setup({
+        container: "kasko-widget-container",
+        key: "KEY",
+        touchpoint: "tp_a3b836dd404a547953a5b8f0dd9ca",
+        mode: "fullscreen",
+        language: "de",
+      });
```

Comment:
Dog and Cat webapps are using `adnr` numbers - is this webapp using it also?

Reply:
Can't see any usage in webapp repo